packages/neon-api/src/provider/dora/class.ts

```diff
@@ -46,6 +55,12 @@ export class Dora implements Provider {
   ): Promise<PastTransaction[]> {
     throw new Error("Method not implemented.");
   }
+  public getAddressAbstracts(
+    address: string,
+    page: number
```

Comment:
Just a suggestion that you might want to have a default page number of 0 to start with?

Reply:
I think that don't exist page zero in api response, I gonna verify but I can use number 1 like default

Reply:
any updates on this? This is pretty important to get right.